tests/test_kernel_operations.py

```diff
@@ -0,0 +1,72 @@
+"""
```

Comment:
Very basic tests for the functions in the ml_kernel_operations.py module but useful as a template for further tests.